res/values/strings.xml

```diff
@@ -882,6 +894,20 @@
     <string name="show_window">Show Window</string>
     <string name="login_socks5_experimental_warning">SOCKS5 support is currently experimental. Please use at your own risk. If you type in an address in the e-mail field, there will be DNS lookup that won\'t get tunneled through SOCKS5.</string>
 
+    <!-- desktop keybindings -->
+    <string name="desktop_keybindings_composer">MessageInput / Composer</string>
```

Comment:
i think, this needs some context for the translators (just add a html-comment above the strings, see other strings with context.